npm-packages/meteor-installer/README.md

```diff
@@ -1,16 +1,54 @@
 ## Meteor Installer
 
-Node.js <=14.x and npm <=6.x is recommended.
+### Recommended Versions
 
-Install Meteor by running:
+- For Meteor 2
+  - Use Node.js 14.x or higher
+  - Use npm 6.x or higher
+- For Meteor 3
+  - Use Node.js 20.x or higher
+  - Use npm 9.x or higher
```

Comment:
```suggestion
- For Meteor 3
  - Use Node.js 20.x
  - Use npm 9.x
```

Has Meteor 3.0 been tested with newer versions of Node.js and npm? If not, we should recommend specific versions.

Reply:
This only applies to the installer, so v20 would be the minimum for v3, soon we will be updating for v22 but that would be bundled into the framework.